src/data/nat/digits.lean

```diff
@@ -414,3 +418,30 @@ begin
   rw of_digits_neg_one at t,
   exact t,
 end
+
+lemma digits_len_le_digits_len_succ (b n : ℕ) : (digits b n).length ≤ (digits b (n + 1)).length :=
+begin
+  cases b,
+  { -- base 0
+    cases n; simp },
+  { cases b,
+    { -- base 1
+      simp },
+    { -- base >= 2
+      apply nat.strong_induction_on n,
+      clear n,
+      intros n IH,
+      cases n,
+      { simp },
+      { repeat { rw digits_add_two_add_one },
+        by_cases hdvd : (b.succ.succ) ∣ (n.succ+1),
+        { rw nat.succ_div_of_dvd hdvd,
+          repeat { rw list.length_cons },
+          rw nat.succ_le_succ_iff,
```

Comment:
```suggestion
        { rw [nat.succ_div_of_dvd hdvd, list.length_cons, list.length_cons, nat.succ_le_succ_iff],
```

(I prefer to avoid using `repeat` when you really mean: do it twice. It usually doesn't save much space, and decreases readability, because the reader now has to reason about how many times the tactic might apply.)

Reply:
(but this is a very minor stylistic point, feel free to ignore)

Reply:
oop I just changed the repeats to rw twice